package.json

```diff
@@ -35,19 +35,19 @@
     "@types/lodash": "^4.14.149",
     "@types/node": "^8.10.59",
     "@types/optimist": "0.0.29",
-    "@types/prettier": "^1.18.3",
+    "@types/prettier": "^1.19.1",
     "coffee-script": "^1.10.0",
     "coffeelint": "^1.15.0",
     "coffeescope2": "^0.4.5",
     "depcheck": "^0.6.7",
     "glob": "^7.1.6",
     "lodash": "^4.17.15",
     "optimist": "^0.6.1",
-    "prettier": "^1.19.1",
-    "tslint": "^5.20.1",
+    "prettier": "^2.0.0",
```

Comment:
prettier 2.0 made a bunch of changes to internal apis, did none of them affect us? https://prettier.io/blog/2020/03/21/2.0.0.html

Reply:
`resin-lint` with/without `--fix` still seem to work fine.

Reply:
The binary is called `balena-lint` now so if `resin-lint` is working you're probably not actually testing your changes

Reply:
that was a force of habit, I'm actually running `/Users/wrboyce/.nvm/versions/node/v12.16.1/bin/balena-lint`.

Reply:
(also it lints typescript 3.8 code, and picks up my changes to `.prettierrc`)

Reply:
Hang on, does it mean that it stops working with Node 8? How about waiting until Node 14 becomes a "current" release, which seems to be [just a month away](https://nodejs.org/en/about/releases/)? `balena-cli` has been supporting Node 8, 10 and 12. When Node 14 becomes current, we could start switching to Node 10, 12 and 14. (Node 14 becomes LTS later in the year.)

Reply:
@pdcastro node 8 has been EOL since 2020-01 and in maintenance only since 2019-01 - we shouldn't actually be using it for dev anywhere and dropping support for it atm is absolutely fine atm - the nodejs developers already have. We can add node 13 for testing if we want but it probably makes sense to skip that and instead add testing for node 14 when it gets released in (scheduled 2020-04) so we make sure we're compatible for when it hits LTS

Reply:
OK, fair enough 👍

Reply:
Also on a sidenote this doesn't stop a project from supporting node 8 (or 6 for that matter) as long as it uses typescript with the right tsconfig target, it just stops being able to develop/run linting on it - but dropping node 6/8 does offer up a lot of benefits for the features/apis that can be used (and is necessary to be able to generate native async/await rather than transpiled and get the benefits from the native version)